### PR TITLE
Update README and docs install page images

### LINKED ISSUE #376: Update `README.md` photo with a more eye-catching file
The images currently showed in the README.md file are in low resolution as well as giving little idea of what differentiates the [`admin`](https://github.com/g3w-suite/g3w-admin) module from the [`client`](https://github.com/g3w-suite/g3w-client) part.

Here, for example, is how they could be replaced:

## Before (v3.4)

immagine

immagine

## After

immagine

### PR DESCRIPTION
**Pull request by wlorenzetti**
Closes: #376

### CODE REVIEW

**Comment by wlorenzetti** [comment]
@Raruto did you review it?

File: README.md [hunk 1]
```diff
@@ -6,10 +6,9 @@ G3W-ADMIN admin server module for G3W-SUITE. (Django 2.2 and Python 3.6, current
 
 G3W-SUITE works with **QGIS-Server LTR versions**.
 
-![Admin GUI](docs/images/install/en/admin.jpg)
+![Admin GUI](docs/images/install/en/admin.png)
 
-![Webgis GUI](docs/images/install/en/client.jpg
-)
+![Webgis GUI](docs/images/install/en/client.png)
```
Comment (Raruto) [inline README.md:11]: @wlorenzetti that image is misleading when shown here as well.

The map viewer (although included here) is developed in another repository anyway: [g3w-client#readme](https://github.com/g3w-suite/g3w-client#readme)

**Review by Raruto — COMMENTED** [review]
@wlorenzetti obviously this pull is also linked to this issue: https://github.com/g3w-suite/g3w-admin/issues/375

As I said, I think it would be better to move the entire `docs` folder into a dedicated repository (trivially also because adding in here additional information on how to contribute to the documentation would increase the readability and complexity of the README file even more ..)

Not to mention all the useless github workflows (e.g. tests, deploy, ...) that are triggered every time someone makes a change to documentation files ...

**Comment by Raruto** [comment]
Superseed by: https://github.com/g3w-suite/g3w-admin/pull/413

**Pull request closed without merging**